### PR TITLE
Copy update on /containers/what-are-containers

### PR DESCRIPTION
**Pull request by minkyngkm**
## Done

- Update assets on `/containers/what-are-containers`

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/containers/what-are-containers
- Be sure to test on mobile, tablet and desktop screen sizes
- Please check page against [copy doc](https://docs.google.com/document/d/1hStuIzY8dDdFEW4Q0TwboOx0_-SB5klk8zRuNIn3V4o/edit#) and resolve comments
- [Demo](https://ubuntu-com-10937.demos.haus/containers/what-are-containers)

## Issue / Card

Fixes [#4638](https://github.com/canonical-web-and-design/web-squad/issues/4638)

## Screenshots

image
image

## Help

[QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html) - [Commit guidelines](https://canonical-web-and-design.github.io/practices/workflow/commit-messages.html)

### CODE REVIEW

**Comment by minkyngkm** [comment]
Currently waiting for an asset to be provided.

**Comment by sowasred2012** [comment]
Code looks good, will approve once we have the fixed asset.

**Comment by sowasred2012** [comment]
@waynecrosby sorry, the new asset font is correct now, but there seems to be a lot aliasing on the whole thing - can you help? https://ubuntu-com-10937.demos.haus/containers/what-are-containers

**Comment by waynecrosby** [comment]
@sowasred2012 I can't see any aliasing, can you take a screengrab to show me, thanks

image

**Comment by sowasred2012** [comment]
@waynecrosby this is what it looks like on Chrome (on Ubuntu), compared to the image in the section above:

Containers vs Virtual Machines:
Screenshot from 2021-11-30 09-16-14

Docker vs LXD: comparing container technologies
Screenshot from 2021-11-30 09-16-26

GIF
Peek 2021-11-30 09-16

**Comment by waynecrosby** [comment]
@sowasred2012 That is really strange. I'm assuming it has been implemented in the exact same way os the one above it?

File: templates/containers/what-are-containers.html [hunk 1]
```diff
@@ -127,6 +127,17 @@ <h2>Containers vs virtual machines</h2>
   <div class="u-fixed-width">
     <h2>Docker vs LXD: comparing container technologies</h2>
     <p>Process containers (for example, Docker) and machine containers (traditional <a href="https://linuxcontainers.org/" class="p-link--external">linux containers</a>) are two types of container technologies that can address different use cases. Process containers typically contain a single application, while machine containers can contain one or more applications as well as their own operating system. Both container technologies share a kernel with the host operating system. </p>
+    <div class="u-hide--small u-align--center">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/4d5fcd9f-What+are+containers.svg",
```
Comment (sowasred2012) [inline templates/containers/what-are-containers.html:132]:
```suggestion
        url="https://assets.ubuntu.com/v1/c06f7eed-What+are+containers+-+outlined.svg",
```

**Comment by sowasred2012** [comment]
@minkyngkm @waynecrosby all looks good now, thanks both!

**Review by sowasred2012 — APPROVED** [review]
LGTM

**Pull request merged**